src/Widgets/FontAwesomeIcon.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
@yemexx1 The use of this widget has been replaced with a static function on the `CottaCush\Yii2\Helpers\Html` class